packages/test/local-server-stress-tests/src/localServerStressHarness.ts

```diff
@@ -913,8 +913,19 @@ async function synchronizeClients(connectedClients: Client[]) {
 							connectedClients[0].container.deltaManager.lastSequenceNumber,
 				)
 			) {
-				resolve();
-				off();
+				// There are some cases where more ops are generated as a result of processing ops.
+				// For example, for ConsensusOrderedCollection, a `complete`/`release` op will be generated
+				// when processing an `acquire` op.
```

Comment:
Technically it's not generated during (synchronous) processing but rather is generated "soon afterwards", which is the source of the challenge.

This approach can't be airtight since the logic a DDS might run async after processing a message is theoretically arbitrary.  This might be reasonable as a best-effort in the shorter term to get the test stood up, but maybe we should log a backlog item to support async reducers so we can use the approach we discussed prior?